web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -139,6 +139,7 @@ export enum DiscoverEventResource {
   ApplicationHttp = 'DISCOVER_RESOURCE_APPLICATION_HTTP',
   ApplicationTcp = 'DISCOVER_RESOURCE_APPLICATION_TCP',
   WindowsDesktop = 'DISCOVER_RESOURCE_WINDOWS_DESKTOP',
+  WindowsDesktopNonAD = 'DISCOVER_RESOURCE_DOC_WINDOWS_DESKTOP_NON_AD',
```

Comment:
I think this needs to correspond to `DiscoverResource` in `usageevents.proto` which itself needs to be deployed to Cloud first.

https://github.com/gravitational/teleport/blob/464edfa2cdc7acc4e8d2b711fef3e9740e21d4d5/api/proto/teleport/usageevents/v1/usageevents.proto#L96-L108

For now I'd just use `WindowsDesktop` and ask on #dev-discover if we want to track these separately, unless it was discussed already.

Reply:
Good catch. We absolutely want to track separately.

Reply:
Thanks for catching that, I've modified the proto and opened PR in cloud

Reply:
@zmb3 @probakowski wdyt about adding a usage event for reaching the end of the discover AD flow as well?

Reply:
I was going to suggest the same. Might as well since we're here.